gnovm/cmd/gno/util.go

```diff
@@ -343,3 +343,19 @@ func prettySize(nb int64) string {
 	}
 	return fmt.Sprintf("%.1f%c", float64(nb)/float64(div), "kMGTPE"[exp])
 }
+
+// isExecutionSuccessfulWithinTimeout checks if the execution of a given exec.Cmd
+// is successful within a specified timeout.
+func isExecutionSuccessfulWithinTimeout(cmd *exec.Cmd, timeout time.Duration) bool {
```

Comment:
I'm on the fence about even having this method.

You only use it to verify the browser was opened -- why? I don't think the browser opening is contingent on this command working correctly. 

You can always print the issue body in the console, regardless if the browser opened or not

Reply:
Removed.